_sass/blocks/explore/_filters.scss

```diff
@@ -15,6 +25,19 @@
     border: 0;
     height: 50px;
   }
+
+  &[aria-expanded=true] {
+    // background-color: $mid-blue;
+    background-color: rgba(198,233,255, 0.85) !important;
+  }
+
+  &.color {
+    background: rgba(198,233,255, 0.85);
```

Comment:
Color literals like `rgba(198, 233, 255, 0.85)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by a single space